app/controllers/api/v1/articles_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
   def create
     article = Article.create(article_params)
     if article.persisted?
-      render json: { message: 'Successfully created' }
+      render json: { message: 'Successfully created', data: article.id }
```

Comment:
I would not use `data here. Why not use `article_id`?

Reply:
changed